### PR TITLE
DPL-890-6 Vue 3 upgrade changes

### PR DESCRIPTION
**Pull request by Sangeetha-Bheeman**
Closes #

#### Changes proposed in this pull request

Involves (breaking)changes to support Vue 3 upgrade
- Upgraded to vue 3.3.4
- Replaced unsupported changes
- Removed and replaced Vue.prototype to support the Store
- Removed createLocalVue
- Removed and replaced Vue.extend
- Removed creation of Vue component instances using new keyword
- Test changes to use mount
- Router changes

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from 'vite'
-import { createVuePlugin as vue } from 'vite-plugin-vue2'
+// import { createVuePlugin as vue } from 'vite-plugin-vue2'
```
[the remark this answers is not in the document]
Comment (Sangeetha-Bheeman) [reply vite.config.js]: Removed the commented line.

**Comment by Sangeetha-Bheeman** [comment]
> All seems sensible. Only one minor comment.
> 
> Could you also look through and remove any scoped styles?
> 
> I think it would be a good idea to look through and improve the documentation but we could roll it into another story.

Yes, the documentation can be improved, for now I just removed the irrelevant ones. I just made a qucik check looking for any scoped styles, and I have already made changes in the past story to move them to tailwind.css and the components don't have any scoped styles inside them.

**Comment by stevieing** [comment]
> > All seems sensible. Only one minor comment.
> > Could you also look through and remove any scoped styles?
> > I think it would be a good idea to look through and improve the documentation but we could roll it into another story.
> 
> Yes, the documentation can be improved, for now I just removed the irrelevant ones.
> 
> I also made a quick check looking for any scoped styles, and I have already made changes in the past story to move them to tailwind.css and the components don't have any scoped styles inside them.

The following components still have style tags. It is probably better to remove them as we no longer want to use them:
- PrintJob.vue
- PageFooter.vue
- Plate.vue
- PlateList.vue
- Row.vue
- Upload.vue
- App.vue
-

**Comment by Sangeetha-Bheeman** [comment]
> > > All seems sensible. Only one minor comment.
> > > Could you also look through and remove any scoped styles?
> > > I think it would be a good idea to look through and improve the documentation but we could roll it into another story.
> > 
> > 
> > Yes, the documentation can be improved, for now I just removed the irrelevant ones.
> > I also made a quick check looking for any scoped styles, and I have already made changes in the past story to move them to tailwind.css and the components don't have any scoped styles inside them.
> 
> The following components still have style tags. It is probably better to remove them as we no longer want to use them:
> 
> * PrintJob.vue
> * PageFooter.vue
> * Plate.vue
> * PlateList.vue
> * Row.vue
> * Upload.vue
> * App.vue

Sure. Removed them.

**Review by stevieing — APPROVED** [review]

**Pull request merged**